CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Change Log
```

Comment:
can you use github issues to keep track of things to do ? and remove this changelog file?

Reply:
Removed